notes/sparse.md

```diff
@@ -42,8 +70,17 @@ $$\textrm{col} = \begin{bmatrix} 0 & 3 & 0 & 1 & 3 & 0 & 2 & 3 & 4 & 2 & 3 & 4\e
 
 $$\textrm{rowptr} = \begin{bmatrix} 0 & 2 & 5 & 9 & 11 & 12 \end{bmatrix}$$
 
+
+(\textrm{rowptr}\\) contains the row offset (array of <span>\\(n + 1\\)</span>)  integers).
+
+\\(\textrm{col}\\) contains column indices (array of <span>\\(nnz\\)</span>)  integers).
+
+\\(\textrm{data}\\) contains non-zero elements (array of <span>\\(nnz\\)</span>)  doubles).
+
 How to interpret: The first two entries of \\(\textrm{rowptr}\\) gives us the elements in the first row. Interval [0, 2) of \\(\textrm{data}\\) and \\(\textrm{col}\\), corresponding to two (data, column) pairs: (1.0, 0) and (2.0, 3), means the first row has 1.0 at column 0 and 2.0 at column 3. The second and third entries of \\(\textrm{rowptr}\\) tells us [2, 5) of \\(\textrm{data}\\) and \\(\textrm{col}\\) corresponds to the second row. The three pairs (3.0, 0), (4.0, 1), (5.0, 3) means in the second row, there is a 3.0 at column 0, a 4.0 at column 1, and a 5.0 at column 3. 
 
+This method does not store the zero elements and provides fast arithmetic operations between sparse matrices, and fast matrix vector product. 
+
 ## CSR Matrix Vector Product Algorithm
 
 The following code snippet performs CSR matrix vector product for square matrices:
```

Comment:
explain this a little more, and maybe move up to be with the rest of the section

Reply:
Also add yourself to the changelog

Reply:
actually, move the changelog to the yaml header